src/sentry/api/base.py

```diff
@@ -62,7 +62,7 @@ class DocSection(Enum):
 class Endpoint(APIView):
     authentication_classes = DEFAULT_AUTHENTICATION
     renderer_classes = (JSONRenderer,)
-    parser_classes = (JSONParser,)
+    parser_classes = (JSONParser, MultiPartParser, FormParser,)
```

Comment:
I'm not sure whether this makes sense, but should we set these new parser_classes only on endpoints that accept files?

Reply:
That can definitely be done, but I don't think tightening that is necessary because DRF will choose the appropriate parser automatically: https://www.django-rest-framework.org/api-guide/parsers/#how-the-parser-is-determined.

If anything I'd even remove `parser_classes` and put those three parsers in `DEFAULT_PARSER_CLASSES`.

But I'm not opinionated / have little experience with DRF. Perhaps someone else could chime in on what to do here?

Reply:
That might make sense too. As far as I can tell these were added back in 2015, and there's no explicit reason they were set like this. I guess my only concern is whether there are potential security problems with allowing these on every endpoint. I can't think of anything in particular that might happen, so maybe what you've got is fine.

Reply:
I'll move them to `DEFAULT_PARSER_CLASSES` so we can place more sources of truth in the conf.